sampleapp/README-Container.md

```diff
@@ -96,12 +97,14 @@ spec:
 
 [Validating AppSpec](https://github.com/cohesity/cohesity-appspec/blob/master/tools/appspecvalidator/README.md)
 
-## Create Tarball
+## Upload the app via DevPortal
 
-Create a tarball consisting of App.json, Docker Image and AppSpec.
-```bash
-tar cvzf view-browser.tar.gz view-browser:latest development/app.json viewbrowser_spec.yaml 
-``` 
+- Login to [DevPortal](devportal.cohesity.com)
```

Comment:
Can you follow up on : how can one sign up on dev portal?

Reply:
Even I'm not sure on this one, but one needs to get dev portal credentials issued to them. My manager, Rajat Gupta may be able to give more context regarding this.

Reply:
In regards to this Sample app specifically, you may want to include that once you have the app installed on a local cluster, you need to have a view created from 'File Services -> Views' as well as how to mount that view for the application to see it. Opening the app shows a 404 page otherwise.